dashboard/gulp/scripts.js

```diff
@@ -80,20 +80,26 @@ function webpackWrapper(watch, test, callback) {
     webpackOptions.devtool = 'inline-source-map';
   }
 
+  var callbackCalled = false;
+
   var webpackChangeHandler = function (err, stats) {
-    if (err) {
-      conf.errorHandler('Webpack')(err);
-    }
     $.util.log(stats.toString({
       colors: $.util.colors.supportsColor,
       chunks: false,
       hash: false,
       version: false
     }));
-    browserSync.reload();
+
+    if (!watch && stats.hasErrors()) {
+      $.util.log("Webpack task failed with errors");
```

Comment:
We already have implementation for an error handler: ``` conf.errorHandler('Webpack')(err);```. 
It is better to revert the old implementation 
https://github.com/eclipse/che/pull/12911/files#diff-b7e557f6e1ded3f027a85e6b93a04243L85
or rework this line with conf.errorHandler

Reply:
I couldn't use `conf.errorHandler('Webpack')(err);.` because there is no err object available. I will change the log to  `$.util.log($.util.colors.red('[Webpack task failed with errors]'));` so that the text will be red.